src/backend/pyproject.toml

```diff
@@ -61,6 +61,7 @@ dependencies = [
     "python-dateutil==2.8.2",
     "requests==2.31.0",
     "sentry-sdk==1.44.1",
+    "stockholm==0.5.7",
```

Comment:
I wonder if we could not use `django-money` instead to prevent to install a new dependency.